Formula/b/blisp.rb

```diff
@@ -0,0 +1,23 @@
+class Blisp < Formula
+  desc "ISP tool & library for Bouffalo Labs RISC-V Microcontrollers and SoCs"
+  homepage "https://github.com/pine64/blisp"
+  url "https://github.com/pine64/blisp.git",
+      tag:      "v0.0.4",
+      revision: "f601b6b965a922b4a9f400d1c8fd946b69708728"
+  license "MIT"
+  head "https://github.com/pine64/blisp.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", "-DBLISP_BUILD_CLI=ON", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    rm Dir["#{lib}/blisp*.h"]
```

Comment:
have the files been correctly installed?

Reply:
Yes

Reply:
At least, when testing on my intel mac mini, it just did

Reply:
the header files were not installed if you rm

Reply:
relates to:
- https://github.com/pine64/blisp/pull/48
- https://github.com/pine64/blisp/pull/49